src/mpl2/README.md

```diff
@@ -108,6 +110,8 @@ place_macro
 | `-macro_name` | The name of a macro of the design. |
 | `-location` | The lower left corner of the macro in microns. |
 | `-orientation` | The orientation according to odb. If nothing is specified, defaults to `R0`.  We only allow `R0`, `MY`, `MX` and `R180`.  |
+| `-dont_snap` | Enforce the exact `-location`. Note that the default behavior of `place_macro` is to snap the macro, that is, tune the entered `-location` so that the signal pins get aligned with the track-grid. |
+| `-check_overlap` | Place macro only if no overlap with other macros is found. |
```

Comment:
I think this should be default and also to have a positve name, so how about `-allow_overlap`?